lib/features/reps_core/reps_admin/reps_admin.install

```diff
@@ -25,3 +25,21 @@ function reps_admin_disable() {
 function reps_admin_uninstall() {
   drupal_set_message(t("The REPS admin feature has been uninstalled."));
 }
+
+/**
+ * Disable and uninstall webform_features module.
```

Comment:
you can regroup everything inside a single hook_update + the comment is not right